aos-vm.yaml

```diff
@@ -178,8 +178,8 @@ components:
         - "../output/fota/%{UNIT_MODEL}-%{UNIT_VERSION}-%{NODE_TYPE}-full-%{BUNDLE_IMAGE_VERSION}.tar"
 
       additional_deps:
-        - "%{YOCTOS_WORK_DIR}/build-%{NODE_TYPE}/tmp/deploy/images/%{MACHINE}/%{AOS_BASE_IMAGE}-%{MACHINE}.ext4"
-        - "%{YOCTOS_WORK_DIR}/build-%{NODE_TYPE}/tmp/deploy/images/%{MACHINE}/aos-image-initramfs-%{MACHINE}.cpio.gz"
+        - "%{YOCTOS_WORK_DIR}/build-%{NODE_TYPE}/tmp/deploy/images/%{MACHINE}/%{AOS_BASE_IMAGE}-%{MACHINE}.rootfs.ext4"
+        - "%{YOCTOS_WORK_DIR}/build-%{NODE_TYPE}/tmp/deploy/images/%{MACHINE}/aos-image-initramfs-%{MACHINE}.rootfs.cpio.gz"
```

Comment:
I doubt we should add rootfs suffix to initramfs images. Please check with @al1img

Reply:
Currently we have yocto 4.0 supported in main
In yocto 4.3 output image name rules has been update 

https://docs.yoctoproject.org/migration-guides/migration-4.3.html#output-file-naming-changes

We agreed with @al1img  to update yaml file, instead of explicitly setting MAGE_NAME_SUFFIX to “” in recipes